example/res/layout/payment_activity.xml

```diff
@@ -1,27 +1,90 @@
 <?xml version="1.0" encoding="utf-8"?>
 <LinearLayout
     xmlns:android="http://schemas.android.com/apk/res/android"
-    android:layout_width="fill_parent"
-    android:layout_height="fill_parent"
+    android:layout_width="match_parent"
```

Comment:
A slightly more standard version than this would be to have something like a `CardEntryView` class and have the base activity only have two things in it, but this is fine for an example app. #nomorefragments